docs/en/administration/sync.md

```diff
@@ -1,4 +1,243 @@
+
+To do a full synchronization, i.e. always synchronize no matter whether the destination files exist or not, you can add option `--force-update` or `-f`. For example, the following command fully synchronizes `movies` directory from [Object Storage A](#bucketA) to [JuiceFS File System](#bucketC).
+
+```shell
+# mount JuiceFS
+sudo juicefs mount -d redis://10.10.0.8:6379/1 /mnt/jfs
+# full synchronization
+juicefs sync --force-update s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/movies/ /mnt/jfs/movies/
+```
+
+### Pattern Matching
+
+The pattern matching function of the `sync` subcommand is similar to that of `rsync`, which allows you to exclude or include a certain classes of files by rules, and synchronize any set of files by combining multiple rules. Now we have these following rules available.
+
+- Patterns ending with `/` will only match directories, otherwise, they will match files, links or devices;
+- Patterns containing `*`, `?` or `[` will match as wildcards, otherwise, they will match as regular strings;
+- `*` matches any non-empty path components until it sees a `/`;
+- `?` matches any single character except `/`;
+- `[` matches a set of characters, for example `[a-z]` or `[[:alpha:]]`;
+- Backslashes can be used to escape characters in wildcard patterns, while in regular string patterns, they match literally;
+- Match recursively using patterns as prefixes.
+
+#### Exclude Directories/Files
+
+Option `--exclude` can be used to exclude patterns. For example, do a full synchronization from [JuiceFS File System](#bucketC) to [Object Storage A](#bucketA), excluding hidden directories and files:
+
+:::note Remark
+Linux treats a directory or a file whose name starts with `.` as hidden.
+:::
+
+```shell
+# mount JuiceFS
+sudo juicefs mount -d redis://10.10.0.8:6379/1 /mnt/jfs
+# full synchronization, excluding hidden directories and files
+juicefs sync --exclude '.*' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/
+```
+
+Repeat this option with different parameters to exclude multiple patterns. For example, exclude all hidden files, `pic/` directory and `4.png` file:
+
+```shell
+juicefs sync --exclude '.*' --exclude 'pic/' --exclude '4.png' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com
+```
+
+#### Include Directories/Files
+
+Option `--include` is for including patterns you don't want to exclude. For example, synchronize only `pic/` and `4.png` and exclude all the others:
+
+```shell
+juicefs sync --include 'pic/' --include '4.png' --exclude '*' /mnt/jfs/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com
+```
+
+:::info NOTICE
+The earlier options have higher priorities than the latter ones. Your `--include` options should appear before `--exclude`. All `--include` options such as `--include 'pic/' --include '4.png'` which appear later than `--exclude '*'` will be ignored.
```

Comment:
```suggestion
The earlier options have higher priorities than the latter ones. Thus, the `--include` options should come before `--exclude`. Otherwise, all the `--include` options such as `--include 'pic/' --include '4.png'` which appear later than `--exclude '*'` will be ignored.
```